scripts/deploy-federation.sh

```diff
@@ -105,13 +112,24 @@ if [[ ! "${USE_LATEST}" ]]; then
   rm -f ${dockerfile_dir}/controller-manager
 fi
 
-kubectl create ns "${PUBLIC_NS}"
+if [[ "${NAMESPACED}" ]]; then
+  if ! kubectl get ns "${NS}" > /dev/null 2>&1; then
+    kubectl create ns "${NS}"
+  fi
+else
+  if ! kubectl get ns "${PUBLIC_NS}" > /dev/null 2>&1; then
+    kubectl create ns "${PUBLIC_NS}"
+  fi
+fi
 
-# Create a permissive rolebinding to allow federation controllers to run.
+# Create a permissive clusterrolebinding to allow federation controllers to run.
 # TODO(marun) Make this more restrictive.
+# TODO(marun) Use a rolebinding when namespaced once the
+# federaatedtypeconfig controller can limit informer scope to the
```

Comment:
federaated

Reply:
This comment is removed in #230, so I'm inclined not to fix it.

Reply:
ok